extensions/gdx-setup/src/com/badlogic/gdx/setup/ExternalExtension.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package com.badlogic.gdx.setup;
+
+import java.util.HashMap;
+
+/**
+ * Created by azakhary on 2/9/2015.
+ */
+public class ExternalExtension {
+
+    private String name;
+    private String description;
+    private String version;
+
+    private HashMap<String, String> dependencies;
```

Comment:
Also, changed and pushed the Hasmap/Map thing :) that's kind of a nasty habit I have to write HashMap everywhere...